### DISCUSSION TITLE
Does the "Active" toggle apply only to auto-downloads or auto-update as well?

### DISCUSSION THREAD

**Discussion by Ethanol6** [SamTV12345/PodFetch#445] [Q&A]
Sometimes I want to keep a podcast updated but I don't want it automatically downloaded. 

What does the "active" toggle actually control exactly?

**Comment by shooftie** [comment]
I am not the developer but as I understand it, flipping this toggle "off" essentially removes that feed from the auto-update register that is described in the global settings.

![Screenshot 2023-11-20 at 09 55 56](https://github.com/SamTV12345/PodFetch/assets/2161844/0a3a85eb-759b-457c-ab94-9721f3d37718)

I think that what you are after is feed-level configuration, which is not yet implemented.

I have started [a discussion](https://github.com/SamTV12345/PodFetch/discussions/424) about it. It might be worth throwing your pennies worth into there, too. @SamTV12345 is very receptive to suggestions and it makes sense to go for the ones that are going to satiate more users.

**Comment by SamTV12345** [comment]
> I am not the developer but as I understand it, flipping this toggle "off" essentially removes that feed from the auto-update register that is described in the global settings.
> 
> ![Screenshot 2023-11-20 at 09 55 56](https://user-images.githubusercontent.com/2161844/284227061-0a3a85eb-759b-457c-ab94-9721f3d37718.png)
> 
> I think that what you are after is feed-level configuration, which is not yet implemented.
> 
> I have started [a discussion](https://github.com/SamTV12345/PodFetch/discussions/424) about it. It might be worth throwing your pennies worth into there, too. @SamTV12345 is very receptive to suggestions and it makes sense to go for the ones that are going to satiate more users.

Exactly. Thanks for answering 😄 . The usecase is that the original podcast has disappeared but you still want to archive the podcast and listen to it.
